docs/content/guides/dagster/using-environment-variables-and-secrets.mdx

```diff
@@ -0,0 +1,199 @@
+---
+title: Using environment variables and secrets | Dagster Docs
+description: Learn how to securely define and access environment variables in your Dagster code.
+---
+
+# Using environment variables and secrets
+
+<EnvironmentVariablesIntro />
+
+---
+
+## Declaring environment variables
+
+How environment variables are declared depends on how your Dagster project is deployed.
+
+### Dagster Cloud
+
+Environment variables can be set a variety of ways in Dagster Cloud, including directly in the UI. Refer to the [Dagster Cloud environment variables guide](/dagster-cloud/developing-testing/environment-variables-and-secrets) for more info.
+
+### Dagster Open Source
+
+Setting environment variables in Dagster Open Source depends on where Dagster is deployed. Refer to the deployment guide for your platform for more info:
+
+<ArticleList>
+  <ArticleListItem
+    title="Amazon Web Services EC2 / ECS"
+    href="/deployment/guides/aws#secrets-management-in-ecs"
+  ></ArticleListItem>
+  <ArticleListItem
+    title="Docker"
+    href="/deployment/guides/docker"
+  ></ArticleListItem>
+  <ArticleListItem
+    title="Kubernetes"
+    href="/deployment/guides/kubernetes/customizing-your-deployment#security"
```

Comment:
would suggest linking here https://docs.dagster.io/deployment/guides/kubernetes/deploying-with-helm#configure-your-user-deployment

Reply:
Works for me! I wasn't sure what the best place was for this.